test-harness/src/main/java/org/sonar/plugins/findbugs/resource/SmapParser.java

```diff
@@ -1,22 +1,3 @@
-/*
```

Comment:
I think so, please refer https://github.com/spotbugs/spotbugs/pull/132#discussion_r102060390

In my understanding this class is licensed by Michael Schierl, not SonarSource.